### PR TITLE
Md/pmp open posts on journal page fix

### LINKED ISSUE #162: Open post on journal page
When the journal gets saved it shows a list of unmatched(open) vouchers on the same company. Those need to be the same as on postmotpost page. Now it is showing all vouchers. Also add Invoice number, kid and match number for those in the list.

### PR DESCRIPTION
**Pull request by mladjo2505**
Closes #162

### CODE REVIEW

**Comment by mladjo2505** [comment]
Should be rebased after #149 is merged into master

**Comment by mladjo2505** [comment]
First round review by @sls-lnluksa. @stoivo and @sls-lnluksa needed for final review.

**Comment by stoivo** [comment]
Since #149 is not merged and will we tested today. We will not QA this one until it it merged.

**Comment by sls-lnluksa** [comment]
@stoivo 
In open vouchers, if we select one of them (Velg), invoice number and date (causes a warning) are not copied in newly created voucher. 
Would you like to fix that in this PR or create new one?

**Comment by stoivo** [comment]
You can fix it is this one, it should be to much work.

**Comment by mladjo2505** [comment]
I added invoice number and voucher date.

**Comment by mladjo2505** [comment]
Rebased to master since #149 has been merged to master

File: modules/journal/model/search.class.php [hunk 1]
```diff
@@ -50,11 +51,14 @@ function search_openpost_accountplan($args)
...
-                            $_showresult .= "<td><a href=\"".$_MY_SELF.'&amp;voucher_VoucherID='.$args['VoucherID'].'&amp;VoucherType='.$args['VoucherType'].'&amp;voucher_JournalID='.$args['JournalID'].'&amp;voucher_AccountPlanID='.$args['AccountPlanID'].'&amp;voucher_AmountOut='.$row->AmountIn.'&amp;voucher_AmountIn='.$row->AmountOut.'&amp;voucher_KID='.$row->KID.'&amp;type='.$args['type']."&amp;action_voucher_update=1\">Velg</a></td>";
+                            $_showresult .= "<td><a href=\"".$_MY_SELF.'&amp;voucher_VoucherID='.$args['VoucherID'].'&amp;VoucherType='.$args['VoucherType'].'&amp;voucher_JournalID='.$args['JournalID'].'&amp;voucher_AccountPlanID='.$args['AccountPlanID'].'&amp;voucher_AmountOut='.$row->AmountIn.'&amp;voucher_AmountIn='.$row->AmountOut.'&amp;voucher_KID='.$row->KID.'&amp;voucher_VoucherDate='.$row->VoucherDate.'&amp;voucher_InvoiceID='.$row->InvoiceID.'&amp;type='.$args['type']."&amp;action_voucher_update=1\">Velg</a></td>";
```
Comment (stoivo) [inline modules/journal/model/search.class.php]: You should not have added voucher_VoucherDate.

Comment (mladjo2505) [inline modules/journal/model/search.class.php]: Ok, I changed this commit and re-pushed to this branch.

**Comment by sls-lnluksa** [comment]
First round review: Nikola Lukovic (sls-lnluksa)
Final review with: @stoivo

**Pull request merged**